students/isabelle.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <title>Isabelle Bennett</title>
+</head>
+<body>
+    <h1>Isabelle Bennett</h1>
+    <p>Hi, I'm Isabelle! I'm a junior in Computer science. </p>
+    <ul>
+      <li><a href="https://www.linkedin.com/in/isabelle-bennett-8972871b6/">Link to LinkedIn profile</a></li>
+      <li><a href="http://github.com">Or a link to your github profile</a></li>
```

Comment:
feel free to change the link text to be your name or something else, also the github link is still pointing to the main page instead of your user.